compiler/rustc_target/src/spec/mod.rs

```diff
@@ -1064,14 +1084,14 @@ impl TargetWarnings {
 #[derive(PartialEq, Clone, Debug)]
 pub struct Target {
     /// Target triple to pass to LLVM.
-    pub llvm_target: String,
+    pub llvm_target: Cow<'static, str>,
```

Comment:
`Cow<'static, T>` is used so much in this file that it make sense to introduce a private alias `type StaticCow<T> = Cow<'static, T>`.